### DISCUSSION TITLE
Question about Release 1.6 RC1

### DISCUSSION THREAD

**Discussion by ssethuma1155** [vmware-tanzu/velero#3646] [Community support Q&A]
I went through the list of changes for Release 1.6 RC1 and was wondering if the fixed for issue: https://github.com/vmware-tanzu/velero/issues/1914  is in it.  I don't see it specifically called out. According to the last update for this issue, it was slated to be part of release 1.6.  Can someone confirm if it is there?

Suren

**Comment by ssethuma1155** [comment]
I download rc1 and unfortunately this does not seem to have been fixed. When trying to set  includeClusterResources: nil (or null) in the restore yaml file, I see an error 'incorrect type. Expected: boolean'. 

Possible that I am not doing this right. Would appreciate some help.

**Reply by carlisia** [reply]
Hi. That fix went into a much earlier release (2019), not v1.6. Let's start with this explanation, and see if this clarifies it or if we need to look into it further:

https://github.com/vmware-tanzu/velero/pull/2118/files#r357284471

**Reply by ssethuma1155** [reply]
OK. The response (on Jan 13) to issue https://github.com/vmware-tanzu/velero/issues/1914 seems to indicate that it was going in release 1.6 .  Is that a different issue?
Also, when I run 'velero restore' from command line with option --include-cluster-resources=nil (or null) it is ignored.  Is this because 'nil' is the default for this option?
